app/presenters/tasks/generic.rb

```diff
@@ -8,14 +8,20 @@ def can_start?
       fulfilled?(self.class::PREVIOUS_TASK)
     end
 
-    def completed?(rec = record, form = self.class::FORM)
-      form.build(rec, application:).valid?
+    def completed?(rec = record, form = associated_form)
+      form ? form.build(rec, application:).valid? : true
     end
 
     private
 
     def record
       application
     end
+
+    def associated_form
+      return unless defined? self.class::FORM
```

Comment:
could also have done this by passing in nil as the form.. but as I said this isn;'t really required